### PR TITLE
Landing page and some stuff

### PR DESCRIPTION
**Pull request by borditamas**
Mivel a Github Pages tud html-t is kezelni, gondoltam a landing page legyen kicsit jóképűbb :)

image

### CODE REVIEW

**Review by Katinka666 — APPROVED** [review]

File: docs/articles/contribute.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Eclipse Contributor Agreement
+
+Since Arrowhead project is governed within the Eclipse Foundation, the authors of any contribution must agree and accept the [Eclipse Contributor Agreement ("ECA")](https://www.eclipse.org/legal/ECA.php).
+
+Technically it means that contributors have to have an [Eclipse Fundation Account](https://accounts.eclipse.org/user/register?destination=user/edit) with which the signed ECA is possible to submit! GitHub Pull Requests could be accepted only when all the authors own an Eclipse Foundation Account with the signed ECA!
```
Comment (rbocsi) [inline docs/articles/contribute.md:5]: have to have => need

File: docs/articles/contribute.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### ECA verification
+
+The existance of the signed ECA is verified based on the authors commits. Your local Git client should hold and push the committs with the same email address like the Eclipse Foundation Account was created with. When the email address is different, than the verification process will fail and the Pull Request is not possible to merge.
```
Comment (rbocsi) [inline docs/articles/contribute.md:9]: existance => existence

File: docs/index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+				<a href="./authentication/index"><div class="sys-group-list-item authentication-color"><p>System Authentication</p></div></a>	
+			</div>
+		</div>
+		<div class="sys-group">
+			<div class="sys-group-title">Support System documents</div>
+			<div class="sys-group-list">
+				<div class="sys-group-list-item"><p>Event Handler</p></div>
+				<div class="sys-group-list-item"><p>Configuration</p></div>
+				<div class="sys-group-list-item"><p>Translation</p></div>
+				<div class="sys-group-list-item"><p>QoS Manager</p></div>
```
Comment (rbocsi) [inline docs/index.html:44]: This not the name of the support system. I don't remember the exact name but it was something like this: Default QoS Evaluator

File: docs/articles/tutorials.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Tutorials
+
+### [How to registery a system?]()
```
Comment (rbocsi) [inline docs/articles/tutorials.md:3]: registery -> register

**Review by rbocsi — CHANGES_REQUESTED** [review]
see inline comments

**Review by rbocsi — APPROVED** [review]

**Pull request merged**